src/utils/UrlUtils.ts

```diff
@@ -59,6 +59,58 @@ export class UrlUtils {
     };
   }
 
+  public static getLinkDestination(origin: string, href: string) {
+    if (!this.urlIsRelative(href)) {
+      return href;
+    }
+    let currentPosition = this.removeAtEnd(this.getRawUrlParameters(origin), origin);
+    if (!href) {
+      return currentPosition;
+    }
+    href.split('/').forEach((urlPart, index) => {
+      if (!urlPart && !index) {
+        currentPosition = this.getFullHostPath(origin) || '';
+      } else if (urlPart === '..') {
```

Comment:
Is the search-api sending back relative urls? If so, what was the reason to not send absolute urls instead?

Needing the client to resolve urls means the api is inherently harder to use. Would to be possible to place the logic at the api layer instead?

Reply:
I had a discussion about this with Olivier and I'll remove this URL resolution and let Search API handle it.

Reply:
Not search API, but ML model should be returning us answers that are usable. Small, but important detail :)

Reply:
Woops, my bad.